### PR TITLE
feat(frontend, sdk): new apr and rewards handling

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

File: packages/frontend/src/components/campaigns/rewards/index.tsx [hunk 1]
```diff
@@ -21,7 +21,9 @@ export function Rewards({ from, to, rewards }: RewardsProps) {
           dayjs.unix(to).diff(dayjs.unix(from), "days", false)
         : 0;
 
-    return (
+    return perDayUsdValue === 0 ? (
+        <Typography>-</Typography>
```
Comment (guerrap) [inline packages/frontend/src/components/campaigns/rewards/index.tsx:25]: We should add `weight="medium"` just like the apr.

File: packages/frontend/src/ui/pool-remote-logo/index.tsx [hunk 2]
```diff
@@ -20,13 +20,19 @@ export function PoolRemoteLogo({
...
+            <RemoteLogo
+                chain={chain}
+                size={size}
+                {...token1}
+                defaultText={token0?.defaultText}
+            />
+            <RemoteLogo
+                chain={chain}
+                size={size}
+                {...token0}
```
Comment (guerrap) [inline packages/frontend/src/ui/pool-remote-logo/index.tsx:33]: Here we are mixing the tokens, also we could use the `PoolRemoteLogo` component for this.

Comment (luzzif) [reply packages/frontend/src/ui/pool-remote-logo/index.tsx]: Wait, we are inside the `PoolRemoteLogo` component itself. Anyway, I've addressed the comment in 16addf02c5a29363d358321ab31bbd09d3976d49

File: packages/frontend/src/components/campaigns/rewards/index.tsx (hunk 1 above)
Comment (luzzif) [reply packages/frontend/src/components/campaigns/rewards/index.tsx]: Addressed in 16addf02c5a29363d358321ab31bbd09d3976d49

**Review by guerrap — APPROVED** [review]

**Pull request merged**